experiments/long_runs/land.jl

```diff
@@ -646,13 +646,64 @@ setup_and_solve_problem(; greet = true);
 # read in diagnostics and make some plots!
 #### ClimaAnalysis ####
 simdir = ClimaAnalysis.SimDir(outdir)
-short_names_2D = ["gpp", "ct", "slw", "si"]
+short_names = [
+    "sif",
+    "ra",
+    "gs",
+    "trans",
+    "crae",
+    "clhf",
+    "cshf",
+    "lai",
+    "msf",
+    "rai",
+    "sai",
+    "gpp",
+    "an",
+    "rd",
+    "vcmax25",
+    "nir",
+    "anir",
+    "rnir",
+    "tnir",
+    "par",
+    "apar",
+    "rpar",
+    "tpar",
+    "lwn",
+    "swn",
+    "soc",
+    "airp",
+    "rain",
+    "lwd",
+    "swd",
+    "snow",
+    "sza",
+    "qsfc",
+    "ws",
+    "infil",
+    "shc",
+    "stc",
+    "swp",
+    "soilrn",
+    "soilrae",
+    "soillhf",
+    "soilshf",
+    "hr",
+    "scd",
+    "scms",
+    "ct",
+    "sco2",
+    "swc",
+    "si",
+    "sie",
+]
 times = 0.0:(60.0 * 60.0 * 12):(60.0 * 60.0 * 24 * 7)
 for t in times
```

Comment:
This is going to plot a lot of stuff. Can we select a subset of variables and times we care about?